api/python/ai/chronon/cli/compile/conf_validator.py

```
@@ -213,6 +218,46 @@
     return False
 
 
+def detect_feature_name_collisions(
```

Pylint (3.3.7)

[refactor] Too many local variables (19/15)

(R0914)